src/target/ChangePlan/Action.php

```diff
@@ -195,7 +206,9 @@ private function getTarget(Command $command): TargetInterface
         if ($target === false) {
             throw new ConstraintException('Target must exist to change its plan');
         }
-        $this->ensureBelongs($target, $command->customer);
+        if ($command->check_belonging === true) {
```

Comment:
```suggestion
        if ($command->check_belonging) {
```

Там булево свойство, этого достаточно